db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+
+Category.create(name: "TV Comedies")
+Category.create(name: "TV Dramas")
+
+Video.create(title: "Monk", description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.", small_cover_url: "/tmp/monk.jpg", large_cover_url: "/tmp/monk_large.jpg", category_id: 1)
```

Comment:
I would recommend styling this in a different way:

``` ruby
category = Category.create!(name: "Dramas")
Video.create!(title: "Monk",
             description: "After the unsolved murder of his wife, Adrian Monk develops obsessive-compulsive disorder, which includes his terror of germs and contamination.",
             small_cover_url: "/tmp/monk.jpg",
             large_cover_url: "/tmp/monk_large.jpg",
             category: category)
```

makes it more readable. And when you look at the videos being repeated then you can use `times` method ;)